Makefile

```diff
@@ -700,5 +706,5 @@ ifeq ($(shell whoami),root)
 endif
 endif
 
-.PHONY: docker gcloud deploy-redirect-site sync-deps sleep-10 proxy-dashboard proxy-prometheus proxy-grafana clean clean-toolchain clean-binaries clean-protos presubmit test test-in-ci vet
+.PHONY: docker gcloud deploy-redirect-site sync-deps sleep-10 proxy-dashboard proxy-prometheus proxy-grafana clean clean-build clean-toolchain clean-archives clean-binaries clean-protos presubmit test test-in-ci vet
```

Comment:
Looks like clean-archives is redundant here as clean-build already cleans the archive folder. Any case of only cleaning up archive without cleaning the build?

Reply:
I added it just for consistency.

Reply:
Oh wait this is a .PHONY target call out. It needs to be here to tell make it's a .PHONY.